rules/windows/execution_command_prompt_connecting_to_the_internet.toml

```
@@ -23,11 +48,65 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Command Prompt Network Connection"
+note = """## Triage and analysis
+
+### Investigating Command Prompt Network Connection
+
+Attackers commonly transfer tooling or malware from external systems into a compromised environment using the command and control channel. However, they can also abuse signed utilities to drop these files.
```

```suggestion
Attackers commonly transfer tooling or malware from external systems into a compromised environment using `cmd.exe` as a command and control channel. However, they can also abuse signed utilities to drop these files.
```
Not sure if this is what you meant, but "using the command and control channel" doesn't seem right

---

That means that attackers can use Command and Control (https://attack.mitre.org/tactics/TA0011/) channels to drop tools into a compromised host.